portality/static/widget/fq_widget_depends_compiled.js

```diff
@@ -1026,12 +1026,12 @@ function dateHistogramShowLess(options, context, facet) {
 }
 
 var licenceMap = {
-    "CC BY" : ["https://doaj.org/static/doaj/images/cc/by.png", "http://creativecommons.org/licenses/by/3.0/"],
-    "CC BY-NC" : ["https://doaj.org/static/doaj/images/cc/by-nc.png", "http://creativecommons.org/licenses/by-nc/3.0/"],
-    "CC BY-NC-ND" : ["https://doaj.org/static/doaj/images/cc/by-nc-nd.png", "http://creativecommons.org/licenses/by-nc-nd/3.0/"],
-    "CC BY-NC-SA" : ["https://doaj.org/static/doaj/images/cc/by-nc-sa.png", "http://creativecommons.org/licenses/by-nc-sa/3.0/"],
-    "CC BY-ND" : ["https://doaj.org/static/doaj/images/cc/by-nd.png", "http://creativecommons.org/licenses/by-nd/3.0/"],
-    "CC BY-SA" : ["https://doaj.org/static/doaj/images/cc/by-sa.png", "http://creativecommons.org/licenses/by-sa/3.0/"]
```

Comment:
see line 3021 - I prepend the doaj_url - this is the same solution I have been using elsewhere for static resources.

I.e. the staging site widget will no longer request assets from the live site.